tests/FakeItEasy.Specs/CreationSpecs.cs

```diff
@@ -425,6 +425,19 @@ public void ClassWithPrivateConstructorCannotBeFaked(Exception exception)
       Could not find a parameterless constructor."));
         }
 
+        [Scenario]
+        public void ClassWithConstructorInParameterCanBeFaked(ClassWithConstructorInParameter fake)
+        {
+            "Given a class with a constructor with an in parameter"
+                .See<ClassWithConstructorInParameter>();
+
+            "When I create a fake of the class"
+                .x(() => fake = this.CreateFake<ClassWithConstructorInParameter>());
+
+            "Then the fake is created"
+                .x(() => fake.Should().BeAFake());
+        }

```

Comment:
Could you please add tests with `ref` and `out` parameters as well? Just for completeness.